src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -532,6 +542,13 @@ public static ImmutableArray<TypedConstant> Encode(TypeSymbol type, TypeSymbol s
                 return names;
             }
 
+            private static bool TryGetNames(TypeSymbol type, ArrayBuilder<string> namesBuilder)
+            {
+                type.VisitType((t, builder, _ignore) => AddNames(t, builder), namesBuilder);
+                Debug.Assert(namesBuilder.Any());
+                return namesBuilder.All(name => name == null);
```

Comment:
Agree with @cston.  The usage of `TryGetNames` above seems wrong at a glance.  It reads that `TryGetNames` succeeded but we are ignoring the result.